### PR TITLE
add protocols to main page

### PR DESCRIPTION
**Pull request by FireFart**
This PR adds TLS1.0, TLS1.1, SSLv2 and SSLv3 to the front page under the client section. Also the Dashboard now checks SSLv3 and SSLv2

### CODE REVIEW

**Comment by FireFart** [comment]
I signed it!

File: domains/misc/badssl.com/index.html [hunk 1]
```diff
@@ -81,6 +81,13 @@ <h2 id="miscellaneous"><span class="emoji">✏️</span>HTTP Input</h2>
     <a href="http://http-credit-card.{{ site.domain }}/" class="bad"><span class="icon"></span>http-credit-card</a>
   </div>
   <div class="group">
+    <h2 id="protocol"><span class="emoji">↔️</span>Protocol</h2>
```
Comment (lgarron) [inline domains/misc/badssl.com/index.html]: Could you put this after the `Key Exchange` section to balance the 2-column layout?

(I like the emoji!)

File: domains/misc/badssl.com/index.html [hunk 2]
```diff
@@ -81,6 +81,13 @@ <h2 id="miscellaneous"><span class="emoji">✏️</span>HTTP Input</h2>
...
+    <a href="https://ssl-v2.{{ site.domain }}/" class="bad"><span class="icon"></span>SSLv2</a>
+    <a href="https://ssl-v3.{{ site.domain }}/" class="bad"><span class="icon"></span>SSLv3</a>
+    <a href="https://tls-v1-0.{{ site.domain }}/" class="dubious"><span class="icon"></span>TLS 1.0</a>
```
Comment (lgarron) [inline domains/misc/badssl.com/index.html]: Nits:
- Link to the port directly.
- Use the subdomain label (`tls-v1-0`) as the link text

File: domains/misc/badssl.com/index.html [hunk 3]
```diff
@@ -81,6 +81,13 @@ <h2 id="miscellaneous"><span class="emoji">✏️</span>HTTP Input</h2>
...
+    <a href="https://tls-v1-1.{{ site.domain }}/" class="dubious"><span class="icon"></span>TLS 1.1</a>
```
Comment (lgarron) [inline domains/misc/badssl.com/index.html]: Same nits.

File: domains/misc/badssl.com/index.html [hunk 4]
```diff
@@ -81,6 +81,13 @@ <h2 id="miscellaneous"><span class="emoji">✏️</span>HTTP Input</h2>
     <a href="http://http-credit-card.{{ site.domain }}/" class="bad"><span class="icon"></span>http-credit-card</a>
   </div>
   <div class="group">
+    <h2 id="protocol"><span class="emoji">↔️</span>Protocol</h2>
+    <a href="https://ssl-v2.{{ site.domain }}/" class="bad"><span class="icon"></span>SSLv2</a>
```
Comment (lgarron) [inline domains/misc/badssl.com/index.html]: Remove SSLv2 and SSLv3 for now.

File: domains/misc/badssl.com/dashboard/sets.js [hunk 5]
```diff
@@ -19,6 +19,8 @@ var sets = [
       {subdomain: "superfish"},
       {subdomain: "edellroot"},
       {subdomain: "dsdtestprovider"},
+      {subdomain: "ssl-v2", port: 1002},
```
Comment (lgarron) [inline domains/misc/badssl.com/dashboard/sets.js]: Remove SSLv2 and SSLv3 for now.

**Comment by lgarron** [comment]
Thanks for the PR!

Unfortunately, our `ssl-v2` and `ssl-v3` implementations aren't working, and disabled for now. Could you take those, or help with #20 to get them working?

**Comment by FireFart** [comment]
@lgarron I added the suggested changes. I also fixed a copy paste error on the `certificate-transparency` id.

I will have a look at #20 later to see if I can come up with a solution for this

**Comment by FireFart** [comment]
@lgarron any chance to get this merged?

**Comment by lgarron** [comment]
Yep, looks good!

**Comment by FireFart** [comment]
awesome thanks!

**Pull request merged**